apps/ledger-live-mobile/src/components/TranslatedError/TranslatedError.tsx

```diff
@@ -28,11 +28,17 @@ export function TranslatedError({ error, field = "title" }: Props): JSX.Element
   // Handling DMK errors
   // Not translated for now
   if (isDmkError(error)) {
-    if (field !== "title") {
-      return <Text>{(error?.originalError as Error)?.message ?? error.message ?? error._tag}</Text>;
-    }
+    const translatedErrorStringId = `errors.${error._tag}.${field}`;
+    if (t(translatedErrorStringId) !== translatedErrorStringId) {
+      return <Text>{t(translatedErrorStringId)}</Text>;
+    } else {
+      const message =
+        field === "title"
+          ? error._tag
+          : (error?.originalError as Error)?.message ?? error.message ?? error._tag;
```

Comment:
```suggestion
const translatedKey = `errors.${error._tag}.${field}`;
const translated = t(translatedKey);

if (translated !== translatedKey) {
  return <Text>{translated}</Text>;
}

const fallbackMessage =
  field === "title"
    ? error._tag
    : (error?.originalError as Error)?.message ?? error.message ?? error._tag;

return <Text>{t(`errors.generic.${field}`, { message: fallbackMessage })}</Text>;
```

I think it has better readability and avoid repeated translation calls.

* `translatedKey` and `translated` make the intent clearer.
* We evaluate `t(translatedKey)` only once to avoid unnecessary function calls.